website/docs/Use-Cases/Task-Oriented-AutoML.md

```diff
@@ -382,6 +382,33 @@ with mlflow.start_run():
 
 ### Extra fit arguments
 
+Extra fit arguments that are needed as one of the argument for model.fit(). The following code shows how you can pass the extra fit argument using the `fit_kwargs_by_estimator` argument in AutoML.fit():
+
+```python
+from flaml import AutoML
+from pandas.tests.extension.conftest import as_frame
+from sklearn.datasets import load_iris
+
+X_train, y_train = load_iris(return_X_y=True, as_frame=as_frame)
+automl = AutoML()
+automl_settings = {
+    "time_budget": 2,
+    "task": "classification",
+    "log_file_name": "test/iris.log",
+    "estimator_list": ["rf"],
+    "max_iter": 0,
+}
+
+automl_settings["fit_kwargs_by_estimator"] = {
```

Comment:
This example is not good because it has only one estimators and sample_weight is not specific to "rf". The point for this example is that we can pass estimator-specific fit_kwargs.

Reply:
This comment is not resolved.